### PR TITLE
Remove field grafanaUrl

### PR DESCRIPTION
**Pull request by janboll**
Field was replaced by grafanaUrls

### CODE REVIEW

File: schemas/app-sre/app-1.yml [hunk 1]
```diff
@@ -383,3 +379,4 @@ required:
 - serviceOwners
 - sopsUrl
 - architectureDocument
+- grafanaUrls
```
Comment (fishi0x01) [inline schemas/app-sre/app-1.yml:382]: last byte not a newline

Comment (janboll) [reply schemas/app-sre/app-1.yml]: fixed

**Review by fishi0x01 — APPROVED** [review]
LGTM! :+1:

**Comment by lechuk47** [comment]
LGTM

**Comment by rporres** [comment]
late to the party, but it looks great

**Pull request merged**